### DISCUSSION TITLE
Figma

### DISCUSSION THREAD

**Discussion by isaacmdt** [imaNNeo/fl_chart#960] [General]
Any chance there is there a Figma file to design with?

**Comment by imaNNeo** [comment]
I think it will be good if people show their designs and other people try to implement them.
What do you think?

**Comment by imaNNeo** [comment]
Did I understand you correctly? Or do you want to contribute to the fl_chart's design?

**Comment by isaacmdt** [comment]


**Reply by imaNNeo** [reply]
Got it. So can you make it open to let everyone use it?

**Reply by DevdouglasGfs** [reply]
Hey, have you made the lib available yet?
